AlastriaID/VerifiableIDNaturalPerson/2022-11/schema.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "EBSI Natural Person Verifiable ID",
+  "description": "Schema of an EBSI Verifiable ID for a natural person",
+  "type": "object",
+  "allOf": [
+    {
+        "$ref": "../../VerifiableAttestation/2022-11/schema.json"
+    },
+    {
+      "properties": {
+        "credentialSubject": {
+          "description": "Property. Defines additional information about the subject that is described by the Verifiable ID",
+          "type": "object",
+          "properties": {
+            "id": {
+              "description": "Property. Defines the DID of the subject that is described by the Verifiable Attestation",
+              "type": "string",
+              "format": "uri"
+            },
+            "familyName": {
+              "description": "Property. Defines current family name(s) of the credential subject",
+              "type": "string"
+            },
+            "firstName": {
+              "description": "Property. Defines current first name(s) of the credential subject",
+              "type": "string"
+            },
+            "dateOfBirth": {
+              "description": "Property. Defines date of birth of the credential subject",
+              "type": "string",
+              "format": "date"
+            },
+            "personalIdentifier": {
+              "description": "Property. Defines the unique national identifier of the credential subject (constructed by the sending Member State in accordance with the technical specifications for the purposes of cross-border identification and which is as persistent as possible in time)",
+              "type": "string"
+            },
+            "nameAndFamilyNameAtBirth": {
```

Comment:
why have you combined Family Name at Birth and First Names at Birth into one?

Reply:
Okay, but in the ARF the PID Attributes are defined separeted. 
![image](https://user-images.githubusercontent.com/17437316/231231030-a14f9f3c-94ea-40a1-a8e6-7b0ecc335b9a.png)

It is for this reason that I have asked you the following question

Reply:
Interesting, I think it will be implemented like that in future public versions of EBSI schemas because the public schemas are from November of 2022.  We could keep it like that or change it according to the ARF but in that case we need to re-define the JSONLD for the examples to be still valid. Maybe @albertoabellagarcia can do that little adjustment?